MapboxCoreNavigation/NavigationService.swift

```diff
@@ -240,7 +240,7 @@ public class MapboxNavigationService: NSObject, NavigationService, DefaultInterf
                                locationSource: NavigationLocationManager? = nil,
                                eventsManagerType: NavigationEventsManager.Type? = nil,
                                simulating simulationMode: SimulationMode = .onPoorGPS,
-                               routerType: Router.Type? = DefaultRouter.self)
+                               routerType: Router.Type? = nil)
```

Comment:
There’s already a fallback further down. This default value was responsible for a warning in client code, because DefaultRouter is internal.